conda_smithy/templates/run_osx_build.sh.tmpl

```diff
@@ -8,6 +8,8 @@ set -xe
 
 MINIFORGE_HOME=${MINIFORGE_HOME:-${HOME}/miniforge3}
 MINIFORGE_HOME=${MINIFORGE_HOME%/} # remove trailing slash
+mkdir -p ${MINIFORGE_HOME}
```

Comment:
This won't work with Miniforge however

The following...

```bash
mkdir tst_mf_home_already_present
bash ~/Downloads/Miniforge3-24.9.0-0-MacOSX-arm64.sh -b -p tst_mf_home_already_present
```

...gives this error...

```
ERROR: File or directory already exists: 'tst_mf_home_already_present'
If you want to update an existing installation, use the -u option.
```

Think this should be moved into the Micromamba block instead